pkg/requirements.pip

```diff
@@ -1,7 +1,8 @@
 leap.common>=0.3.7
+leap.soledad.client>=0.6
```

Comment:
@kalikaneko told me to keep it with 0.6 and add 0.7 to the *VERSION_COMPAT* so setup.py install keeps working. I think the normal procedure is to bump the requirements when the new release comes out.